Staff Procedure for incident handling.md

```diff
@@ -0,0 +1,65 @@
+This procedure has been adopted from the Ada Initiative's guide titled "[Conference anti-harassment/Responding to Reports](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports)”.
+
+-------------------------------------------------------------------------------------------------
+Be sure to have a good understanding of our Code of Conduct policy, which can be found here: [code of conduct policy](code_of_conduct.md)
+
+Also have a good understanding of what is expected from an attendee that wants to report a harassment incident. These guidelines can be found [here](Attendee%20Procedure%20for%20incident%20handling.md)
+
+Try to get as much of the incident in written form by the reporter. If you cannot, transcribe it yourself as it was told to you. The important information to gather include the following:
+
+ - Identifying information (name/badge number) of the participant doing the harassing
```

Comment:
Same comment about badge number

Reply:
Same answer, we don't have it in the plan right now, but may be worth adding.